### PR TITLE
kernelci.org: Add advisory board members

### PR DESCRIPTION
**Pull request by patersonc**
Some information still needs adding for the members, and the list may not be complete.

### CODE REVIEW

File: kernelci.org/content/en/docs/team/board.md [hunk 1]
```diff
@@ -6,11 +6,29 @@ description: "Representatives from each member organization"
...
+
+There is also a Program Manager role to help coordinating things with the wider
+team.
+
+As of today, the Advisory Board is composed of the members listed below with
+their respective roles, email address and IRC nicknames:
+
+* [Guillaume Tucker](mailto:<guillaume.tucker@collabora.com>) (Collabora, Chair) - `gtucker`
+* [Chris Paterson](mailto:<chris.paterson2@renesas.com>) (CIP, Treasurer) - `patersonc`
+* [Gustavo Padovan](mailto:<gustavo.padovan@collabora.com>) (BayLibre, Program Manager) - `???`
```
Comment (gctucker) [inline kernelci.org/content/en/docs/team/board.md:23]:
```suggestion
* [Gustavo Padovan](mailto:<gustavo.padovan@collabora.com>) (Collabora, Project Manager) - `padovan`
```
We may also add the PM and chair role descriptions somewhere in the docs.

File: kernelci.org/content/en/docs/team/board.md [hunk 2]
```diff
@@ -6,11 +6,29 @@ description: "Representatives from each member organization"
...
+* [Don Zickus](mailto:<dzickus@redhat.com>) (Redhat) - `???`
+* [Guy Lunardi](mailto:<guy.lunardi@collabora.com>) (Collabora) - `???`
```
Comment (gctucker) [inline kernelci.org/content/en/docs/team/board.md:25]:
```suggestion
* [Guy Lunardi](mailto:<guy.lunardi@collabora.com>) (Collabora, Marketing) - `glunardi`
```

File: kernelci.org/content/en/docs/team/board.md [hunk 3]
```diff
@@ -6,11 +6,29 @@ description: "Representatives from each member organization"
...
+* [Guenter Roeck](mailto:<groeck@google.com>) (Google) - `???`
```
Comment (gctucker) [inline kernelci.org/content/en/docs/team/board.md:26]: Guenter is not on IRC

File: kernelci.org/content/en/docs/team/board.md [hunk 4]
```diff
@@ -6,11 +6,29 @@ description: "Representatives from each member organization"
...
+* [Greg Kroah-Hartman](mailto:<gregkh@linuxfoundation.org>) (Linux Foundation) - `???`
```
Comment (gctucker) [inline kernelci.org/content/en/docs/team/board.md:27]:
```suggestion
* [Greg Kroah-Hartman](mailto:<gregkh@linuxfoundation.org>) (Linux Foundation) - `gregkh`
```

File: kernelci.org/content/en/docs/team/board.md [hunk 5]
```diff
@@ -6,11 +6,29 @@ description: "Representatives from each member organization"
...
+* [Kevin Hilman](mailto:<khilman@baylibre.com>) (BayLibre) - `khilman`
```
Comment (gctucker) [inline kernelci.org/content/en/docs/team/board.md:28]:
```suggestion
* [Kevin Hilman](mailto:<khilman@baylibre.com>) (BayLibre, General Members Representative) - `khilman`
```

File: kernelci.org/content/en/docs/team/board.md [hunk 6]
```diff
@@ -6,11 +6,29 @@ description: "Representatives from each member organization"
...
+* [KY Srinivasan](mailto:<kys@microsoft.com>) (Microsoft) - `???`
+* [Veronika Kabatov](mailto:<vkabatov@redhat.com>) (Redhat) - `???`
```
Comment (gctucker) [inline kernelci.org/content/en/docs/team/board.md:30]:
```suggestion
* [Veronika Kabatov](mailto:<vkabatov@redhat.com>) (Redhat) - `vkabatova`
```

File: kernelci.org/content/en/docs/team/board.md [hunk 7]
```diff
@@ -6,11 +6,29 @@ description: "Representatives from each member organization"
...
 The Advisory Board (AB) is composed of one representative from each premium
-member organization.  The board elects one Chair and one Treasurer each year.
-A budget is agreed each year and voted by the board.
+member organization, one representative for every five general member
+organizations (maximum of three) and the TSC Chair.
+
+The board elects one Chair and one Treasurer each year. A budget is agreed each
+year and voted by the board.
+
+There is also a Program Manager role to help coordinating things with the wider
```
Comment (gctucker) [inline kernelci.org/content/en/docs/team/board.md:15]:
```suggestion
There is also a Project Manager role to help coordinating things with the wider
```
Kevin's role was called Program Manager but responsibilities have now been split a new Project Manager role and the Chair.

File: kernelci.org/content/en/docs/team/board.md [hunk 8]
```diff
@@ -6,11 +6,29 @@ description: "Representatives from each member organization"
...
+team.
+
+As of today, the Advisory Board is composed of the members listed below with
+their respective roles, email address and IRC nicknames:
+
+* [Guillaume Tucker](mailto:<guillaume.tucker@collabora.com>) (Collabora, Chair) - `gtucker`
```
Comment (gctucker) [inline kernelci.org/content/en/docs/team/board.md:21]:
```suggestion
* [Guillaume Tucker](mailto:<guillaume.tucker@collabora.com>) (Collabora, Chair and Acting TSC Chair) - `gtucker`
```

**Comment by gctucker** [comment]
@patersonc When this PR is ready you can "merge" your commits into one with interactive rebase and force-push, that will also fix the DCO check.  It will get deployed on staging regardless so no need to do this right now.

**Comment by patersonc** [comment]
> @patersonc When this PR is ready you can "merge" your commits into one with interactive rebase and force-push, that will also fix the DCO check. It will get deployed on staging regardless so no need to do this right now.

Ha way ahead of you :P

**Comment by gctucker** [comment]
Now on staging: https://static.staging.kernelci.org/docs/team/board/

**Comment by gregkh** [comment]
LGTM!

**Comment by padovan** [comment]
Ack!

**Pull request merged**